src/styles/blocks/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
+  font-style: normal;
+  padding: 16px;
+  width: 200px;
+  background: #fff;
+  border-radius: 5px;
+  border: 1px solid $border;
+  display: flex;
+  flex-direction: column;
+  &__image {
+    box-sizing: content-box;
+    height: 134px;
+    width: 160px;
+    padding-top: 16px;
+    align-self: center;
+  }
+  
+  &__product-name {
+    padding-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    color: $text-color;
+    line-height: 18px;
+  }
+  
+  &__code {
+    padding-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $inactive-text-color;
+  }
+  
+  &__reviews {
+    padding-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-top: 24px;
+  }
+  
+  &__price-text {
+    line-height: 18px;
+    font-size: 12px;
+    color: $text-color;
+  }
+  
+  &__price-sum {
+    line-height: 18px;
+    font-weight: 700;
+    font-size: 16px;
+    color: $text-color;
+  }
+  
+  &__button {
+    margin-top: 16px;
+  }
+}
+
+.reviews {
+  font-size: 10px;
+  color: $inactive-text-color;
+}
+
+.button {
+  box-sizing: border-box;
+  text-transform: uppercase;
+  width: 100%;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 5px;
+  font-weight: 700;
+  font-size: 14px;
+  color: #fff;
+  text-decoration: none;
+  background: $blue-color;
+}
+
+.button:hover {
+  background-color: #fff;
+  border: 1px solid $blue-color;
```

Comment:
don't add border on hover, add transparent one on default state and change it's color on hover